### ISSUE TITLE
multiplicity models don't work with SQ (Trac #1006)

### ISSUE THREAD

**Issue by butlerpd** [SasView/sasview#1075]
Model with layer choices (e.g. onion, core_multi_shell, etc.) work fine as simple form factors.  However when multiplying by SQ the n layer dropdown no longer controls the number of layers and the default of 10 layers is always given. 

Steve King says this seems to have been introduced in v. 4.0.1

Migrated from http://trac.sasview.org/ticket/1006
```json
{
    "status": "closed",
    "changetime": "2017-11-02T19:48:58",
    "_ts": "2017-11-02 19:48:58.795530+00:00",
    "description": "Model with layer choices (e.g. onion, core_multi_shell, etc.) work fine as simple form factors.  However when multiplying by SQ the n layer dropdown no longer controls the number of layers and the default of 10 layers is always given. \n\nSteve King says this seems to have been introduced in v. 4.0.1",
    "reporter": "butler",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Bug Fixing",
    "time": "2017-10-03T17:05:00",
    "component": "SasView",
    "summary": "multiplicity models don't work with SQ",
    "priority": "critical",
    "keywords": "",
    "milestone": "SasView 4.2.0",
    "owner": "GitHub <noreply@github.com>",
    "type": "defect"
}
```

**Comment by smk78** [comment]
Trac update at `2017/10/03 17:26:48`: **smk78** commented:

After 4.0.1

**Comment by gonzalezma** [comment]
Trac update at `2017/10/30 15:26:51`: 
* **gonzalezm** commented:

Reopen ticket because the SLD profile is not working with the structure factor. Fix will follow.


* **gonzalezm** changed resolution from "fixed" to ""
* **gonzalezm** changed status from "closed" to "reopened"

**Comment by gonzalezma** [comment]
Trac update at `2017/10/30 15:38:30`: 
* **gonzalezm** commented:

In changeset 439ffcd5cba3be56b1d2047b6394dfa9cbcda212:

```text
#!CommitTicketReference repository="sasmodels" revision="439ffcd5cba3be56b1d2047b6394dfa9cbcda212"
Fixes #1075
```


* **gonzalezm** changed resolution from "" to "fixed"
* **gonzalezm** changed status from "reopened" to "closed"
